zellij-client/src/lib.rs

```diff
@@ -145,7 +146,22 @@ pub fn start_client(
     config.env.set_vars();
 
     let palette = config.themes.clone().map_or_else(
-        || os_input.load_palette(),
```

Comment:
If someone has a theme configured in their configuration file, they wont get the benefits of this feature.